sdcm/tester.py

```diff
@@ -782,19 +783,34 @@ def init_resources(self, loader_info=None, db_info=None,
         elif cluster_backend == 'baremetal':
             self.get_cluster_baremetal()
 
-        seeds_num = self.params.get('seeds_num', default=1)
-        if self.params.get('instance_provision') == 'mixed':
-            for node in self.db_cluster.nodes:
-                if seeds_num == 0:
-                    break
-                if node.is_spot:
-                    node.is_seed = True
-                    seeds_num -= 1
+    def set_seeds(self):
+        seeds_selector = self.params.get('seeds_selector')
+        assert seeds_selector in ['reflector', 'random', 'first'], 'Unknown seed selector. ' \
```

Comment:
this should be verified in SCT config with `choices`

Reply:
done